packages/opentelemetry-web/test/StackContextManager.test.ts

```diff
@@ -0,0 +1,198 @@
+/*!
```

Comment:
could you just rename file ?

Reply:
It is a moved file but the git change detection isn't smart enough.